modules/gapi/include/opencv2/gapi/cpu/gcpukernel.hpp

```diff
@@ -45,11 +45,7 @@ namespace cpu
      *
      * Backends are usually "back boxes" for G-API users -- on the API
```

Comment:
While here, can you please fix this typo? "back boxes" -> "black boxes"

Reply:
Fix